src/styles/libs/docz.scss

```diff
@@ -34,3 +34,27 @@ div[data-testid='main-container'] {
     color: var(--theme-ui-colors-props-highlight, #1fb6ff);
   }
 }
+
+// --------------------------------------------------------
+
+.colors {
```

Comment:
Being that this is in the global styles file, this feels a little too generic. I'd either:

1. Rename it to something more specific, like `swatch-grid` or `color-grid` or something like that, or
2. Move the styles into the colors directory and load from the note.

If you go with the second option, you may want to consider making it a module. That could be overkill, but I'll let you explore.